src/harness/harnessUtils.ts

```diff
@@ -1,5 +1,6 @@
 import * as ts from "./_namespaces/ts";
 import * as Harness from "./_namespaces/Harness";
+import { isIdentifier } from "./_namespaces/ts";

```

Comment:
Not sure if auto-import added this, but `ts` is already imported here.

(Would love to get the direct import stuff in but that's another PR.)